config/db.config.js

```diff
@@ -0,0 +1,47 @@
+export const HOST = 'localhost';
+export const USER = 'postgres';
+export const PASSWORD = 'Barbaris179$';
+export const DB = 'postgres';
```

Comment:
It's better to store in env variables. see: https://www.npmjs.com/package/dotenv

Reply:
https://nodejs.dev/en/learn/how-to-read-environment-variables-from-nodejs/